core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/ExceptionListener.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.stitch.core.services.mongodb.remote;
+
+import org.bson.BsonValue;
+
+/**
+ * ExceptionListener receives non-network related errors that occur.
```

Comment:
[q] are we 100% sure this is only for non-network errors? I could see a corrupted event over the network causing an error here.

Reply:
ah I see you just copied this from the previous ErrorListener. let's leave for now but we can discuss offline

Reply:
discussed offline